### ISSUE TITLE
Change the name of Sf2 Player to Soundfont Player

### ISSUE THREAD

**Issue by Rami-Slicer** [LMMS/lmms#5361]
Since Sf2 Player can now use .sf3 files, it seems a bit misleading for it's name to still be Sf2 Player instead of something like "Soundfont Player".

**Comment by Spekular** [comment]
Relevant: In the past, I believe there's been discussion about merging the
SF2/GUS/GIG players into one plugin (UI wise), which would also solve this
problem.

**Comment by mirk0dex** [comment]
Add the "Good first issue" label

**Comment by hossanf** [comment]
Hi! 
 I'm a first-time contributor and would love to give this issue a try, is it still available?

**Comment by Cameron-Kuykendall** [comment]
Hey there :),

Would anyone happen to know if this has been solved?

**Comment by regulus79** [comment]
I don't believe so. #7760 was opened, but it appears the author has not responded in 5 months. If you would like to have a try at it, feel free!

**Comment by aliign** [comment]
Hi. I created a commit for this that addresses the issues with the other commit (outside of changing the artwork).
[#8057 ](https://github.com/LMMS/lmms/pull/8072)
